### ISSUE TITLE
Images do not open when clicked

### ISSUE THREAD

**Issue by aaron404** [sindresorhus/caprine#2197]
Clicking an image in chat no longer opens a larger version to view, only the thumbnail in the chat is visible. Using the snap version 2.59.3 on Ubuntu 24.04

**Comment by tjaska** [comment]
Same issue. Clicking on an image in chat does nothing. Restarting app not helping. 

Caprine v2.60.1. From PopShop, flatpak version.

System info
```
OS: Pop!_OS 22.04 LTS x86_64 
Host: MS-7B17 2.0 
Kernel: 6.9.3-76060903-generic 
Uptime: 3 hours, 40 mins 
Packages: 2447 (dpkg), 47 (flatpak), 8 (snap) 
Shell: bash 5.1.16 
Resolution: 1920x1080, 3440x1440 
DE: GNOME 42.9 
WM: Mutter 
WM Theme: Pop-dark 
Theme: Pop-dark [GTK2/3] 
Icons: Pop [GTK2/3] 
Terminal: gnome-terminal 
CPU: Intel i7-8700K (12) @ 4.700GHz 
GPU: NVIDIA GeForce RTX 2080 Ti Rev. A 
GPU: Intel CoffeeLake-S GT2 [UHD Graphics 630] 
Memory: 16419MiB / 31959MiB 
```

**Comment by mquevill** [comment]
I believe this should be fixed in [v2.60.3](https://github.com/sindresorhus/caprine/releases/tag/v2.60.3). If you're still having this issue with that version, feel free to reopen this issue.

**Comment by MordorMinion** [comment]
I'm having the same issue with Caprine on Windows 11. Only noticed it start in the last few days. 
Version 2.60.3. 
Windows 11 24H2
